docs/features/kubernetes.md

```diff
@@ -1,6 +1,7 @@
 ---
-id: kubernetes-in-backstage
```

Comment:
No `kubernetes/overview.md`, hoping for more content eventually?

Reply:
That's what I did initially, but it will make the URL ->

https://backstage.io/docs/features/kubernetes/overview

As of now, it's https://backstage.io/docs/features/kubernetes consider #4037.

@AlleyWei Should https://backstage.io/docs/features/kubernetes/overview work for you? Given we might want to add more sub-pages like `/kubernetes/configuration` for example.

(On side note I am thinking of adding a feature to the microsite for these good-looking urls to work e.g. `/docs/features/techdocs`, `/docs/features/kubernetes` even when we add more sub-pages later and move the main content to an `overview.md`)

Reply:
that updated URL with /overview works for me.

Reply:
Cool! Will use that to allow future additions.

Reply:
Done! URL is now https://backstage.io/docs/features/kubernetes/overview